pkg/virt-controller/watch/clone/clone.go

```diff
@@ -259,6 +260,24 @@ func (ctrl *VMCloneController) syncSourceVMTargetVM(key string, source *k6tv1.Vi
 		ctrl.logAndRecord(vmClone, TargetVMCreated, fmt.Sprintf("created target VM %s for clone %s", targetVMInfo.Name, vmClone.Name))
 		syncInfo.targetVMCreated = true
 
+		targetVM, ok := targetVMObj.(*k6tv1.VirtualMachine)
+		if !ok {
+			return addErrorToSyncInfo(syncInfo, fmt.Errorf("VM object from vm informer cannot be converted to *VirtualMachine: %v", err))
+		}
+
+		const vmNotReadyErrPattern = "target VM %s is still not ready - %s %s still in progress"
+		if targetVM.Status.SnapshotInProgress != nil {
+			logger.V(defaultVerbosityLevel).Infof(vmNotReadyErrPattern, targetVM.Name, "snapshot", *targetVM.Status.SnapshotInProgress)
+			return syncInfo
+		}
+		if targetVM.Status.RestoreInProgress != nil {
+			logger.V(defaultVerbosityLevel).Infof(vmNotReadyErrPattern, targetVM.Name, "restore", *targetVM.Status.RestoreInProgress)
+			return syncInfo
+		}
+
+		ctrl.logAndRecord(vmClone, TargetVMReady, fmt.Sprintf("target VM %s for clone %s is ready", targetVMInfo.Name, vmClone.Name))
+		syncInfo.targetVMCreated = true
```

Comment:
this should have been targetVMReady right?

Reply:
Absolutely